src/albert/utils/patches.py

```diff
@@ -1,33 +1,412 @@
-from enum import Enum
-from typing import Any
+from copy import deepcopy
 
 from pydantic import Field
 
-from albert.utils.types import BaseAlbertModel
+from albert.resources.data_templates import DataTemplate
+from albert.resources.parameter_groups import (
+    EnumValidationValue,
+    ParameterGroup,
+    ValueValidation,
+)
+from albert.utils.patch_types import PGPatchDatum
 
 
-class PatchOperation(str, Enum):
-    ADD = "add"
-    UPDATE = "update"
-    DELETE = "delete"
+class GeneralPatchDatum(PGPatchDatum):
```

Comment:
Why is this one special and not part of `patch_types.py`?

Reply:
I missed it; good catch!